.github/workflows/contrib-openai.yml

```diff
@@ -217,3 +217,40 @@ jobs:
         with:
           file: ./coverage.xml
           flags: unittests
+  GraphGroupChat:
```

Comment:
To run this test, please change "pull_request_target" into "pull_request" temporarily. After the test passed, change it back.

Reply:
Done.